src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -30,7 +30,7 @@ export const ArtworkApp: React.SFC<Props> = props => {
       <Row>
         <Col sm={8}>
           <ArtworkBanner artwork={props.artwork} />
-          <Spacer mb={4} />
+          <Spacer mb={[2.5, 4]} />
```

Comment:
We should stick with our design system theme -- There's no key for `2.5` which will introduce drift: https://github.com/artsy/palette/blob/master/src/Theme.tsx#L115-L137